src/_kw

```diff
@@ -24,6 +25,18 @@ _kw() {
   fi
 }
 
+_kw_backup() {
+  local force=
+  if (( words[(I)-r|--restore] )); then
```

Comment:
Any specific reason to use this `((` instead of `[[`?

Reply:
At first, I used this approach because the completion functions I was using as reference (the git ones) did the check this way and I could not make it work with `[[` instead of `((`.

After trying a bit, I came up with an alternative way that uses `[[`:
```bash
if [[ "${words[@]}" =~ ' -r | --restore' ]]; then
  ...
fi
```

This approach has the drawback of being more verbose (there are some checks that consider many options) and of using a not so reliable "work around" of transforming the `words` array in a string and comparing it with a regex expression. Fwiu, the former way did this in a efficient and elegant manner, however that is just a hunch and I cannot confirm this.

With those things in mind, what approach should I choose?